.github/workflows/tests.yaml

```diff
@@ -59,9 +59,38 @@ jobs:
     - name: Run PHPCS Code Style Checker
       run: dev/sh/style
 
+  docfx:
+    name: "Run DocFX tests"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.8'
+    - run: pip install gcp-docuploader
```

Comment:
I think we many need to change this to use hashes, to prevent a pip install vulnerability. Same with below in the `Dockerfile`

Reply:
Because this code is not executed while generating artifacts delivered to users (library packages or docs), it's a much less severe vulnerability.